### ISSUE TITLE
[questions/qa-date-format]  Date formats missing JS locales

### ISSUE THREAD

**Issue by r12a** [w3c/i18n-drafts#223]
[source] (https://www.w3.org/International/questions/qa-date-format#opt3) [en]

The article doesn't contain any information about generating date formats using client-side javascript.

**Comment by r12a** [comment]
Actually, we should also add any relevant information from recent additions to HTML.

**Comment by xfq** [comment]
Yes, mentioning the `to*String` methods on `Date` and options for the `Intl.DateTimeFormat` object is probably useful in this article.

re HTML, do you mean `input type="date/datetime-local/..."`? For example, putting a `pattern` attribute on the date input? Or maybe also the `datetime` attribute of the `ins/del` elements?
